### PR TITLE
[Fix] Support value type of different size for pipeline lookup kernel.

### PR DESCRIPTION
**Pull request by jiashuy**

### CODE REVIEW

File: include/merlin/core_kernels/kernel_utils.cuh [hunk 1]
```diff
@@ -49,6 +49,29 @@ __forceinline__ __device__ uint8_t get_digest(const K& key) {
   return static_cast<uint8_t>(hashed_key >> 32);
 }
 
+template <typename ElementType>
```
Comment (rhdong) [inline include/merlin/core_kernels/kernel_utils.cuh:52]: Possibly we can directly use the `int` var as the templete arg and used the `sizeof(VecV)` to fill it.

**Review by Lifann — APPROVED** [review]
Passed correctness check. LGTM

**Pull request merged**